src/interface/sparse_pack.hpp

```diff
@@ -142,7 +143,23 @@ class SparsePack : public SparsePackBase {
     // accessed on device via instance of types in the type list Ts...
     // The pack will be created and accessible on the device
     template <class T>
-    SparsePack GetPack(T *pmd, const std::vector<bool> &include_block = {}) const {
+    SparsePack GetPack(T *pmd, std::vector<bool> include_block = {},
+                       bool only_fine_two_level_composite_blocks = true) const {
+      // If this is a composite grid MeshData object, only include blocks on
+      // the finer level
```

Comment:
and if `only_fine_two_level_composit_blocks` is set

Reply:
fixed.